tutorials/plot_receptive_field.py

```diff
@@ -190,7 +189,7 @@
 scores = np.zeros_like(alphas)
 models = []
 for ii, alpha in enumerate(alphas):
-    rf = ReceptiveField(tmin, tmax, sfreq, freqs, estimator=Ridge(alpha))
+    rf = ReceptiveField(tmin, tmax, sfreq, freqs, estimator=alpha)
```

Comment:
maybe this is where we show off the laplacian as well? I think we should highlight it somehow as most people won't really know what this is. Actually maybe we need a different section of this tutorial for this, or a short example that compares vanilla ridge + laplacian.